__tests__/authentication.test.js

```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 import { createStore } from 'redux';
 import { Provider as ReduxProvider } from 'react-redux';
 import MockAdapter from 'axios-mock-adapter';
-import { render, fireEvent, waitFor } from 'react-native-testing-library';
+import { render, fireEvent, waitFor } from '@testing-library/react-native';
```

Comment:
react-native-testing-library est déprécié :
https://callstack.github.io/react-native-testing-library/docs/migration-v7/#guide-for-react-native-testing-library-users